Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/Weights.h

```diff
@@ -64,7 +64,7 @@ struct Cotangent_value_Meyer_impl
     if(divider == 0 /*|| divider != divider*/) 
     {
       CGAL::collinear(get(ppmap, v0), get(ppmap, v1), get(ppmap, v2)) ? 
-        CGAL_warning_msg(false, "Infinite Cotangent value with degenerate triangle!") :
+        CGAL_warning_msg(false, "Infinite Cotangent value with degenerate triangle!");
```

Comment:
Now there is a compilation error! We are here in the middle of the ternary operator:
```c++
    condition ? something() : something_else()
```